hs_core/templates/pages/genericresource.html

```diff
@@ -2644,7 +2644,8 @@ <h3 style="color: red" id="file-type-error"></h3>
                         json_response = JSON.parse(result);
                         if (json_response.status === 'success')
                         {
-                            $form.find("button").hide();
+                            $(document).trigger(jQuery.Event("submit-success"));
```

Comment:
You can pass the name of the event directly to `trigger()` and skip `jQuery.Event()`